pkg/cloudprovider/provider/openstack/provider_test.go

```diff
@@ -21,6 +21,8 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/kubermatic/machine-controller/pkg/apis/cluster/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
please run [gimps](https://github.com/xrstf/gimps) to follow our import grouping scheme.

Reply:
@kron4eg  done :)